worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -1195,8 +1198,10 @@ class PushDataHandler(val workerSource: WorkerSource) extends BaseMessageHandler
     val diskInfo = workerInfo.diskInfos
       .get(fileWriter.flusher.asInstanceOf[LocalFlusher].mountPoint)
 
+    val minimumUsableSize =
```

Comment:
Hi @SteNicholas , `DiskUtils.getMinimumUsableSize` is costly, IMO we'd better to cache `minimumUsableSize` in a variable instead of calculate for every pushdata. cc @FMX